bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/i18n/luxtronikheatpump_de.properties

```diff
@@ -240,6 +240,7 @@ channel-type.luxtronikheatpump.temperatureCondensation.label = Kondensationstemp
 channel-type.luxtronikheatpump.temperatureExhaustAir.label = Ablufttemp.
```

Comment:
Please remove this file, as translations are made available by crowdin. Only the defualt translation file should be in the PR.